scooby/__init__.py

```diff
@@ -40,7 +40,7 @@
 ]
 
 
-__author__ = 'Dieter Werthmüller & Bane Sullivan'
+__author__ = 'Dieter Werthmüller, Bane Sullivan, & Alex Kaszynski'
```

Comment:
Just thinking out loud, maybe looking into the future. What about 

'Dieter Werthmüller, Bane Sullivan, Alex Kaszynski, and contributors'

or

'Dieter, Bane, Alex, and contributors'

Reply:
I'm up for that

Reply:
```suggestion
__author__ = 'Dieter Werthmüller, Bane Sullivan, Alex Kaszynski, and contributors'
```